src/views/Settings.vue

```diff
@@ -519,7 +519,7 @@ export default defineComponent({
     async setEComStore(event: any) {
       // not updating the ecomstore when the current value in vuex state and selected value are same
       // or when an empty value is given (on logout)
-      if (this.currentEComStore.productStoreId === event.detail.value || !event.detail.value) {
+      if (this.currentEComStore.productStoreId === event.detail.value) {
```

Comment:
Need to find some other way to fix this, as removing the above check will result in error on logout(as added in comment)